tsdb/head_bench_test.go

```diff
@@ -79,6 +86,71 @@ func BenchmarkHeadStripeSeriesCreate_PreCreationFailure(b *testing.B) {
 	}
 }
 
+func BenchmarkHead_WalCommitWithVaryingSeriesSamplesExemplarsAndHistograms(b *testing.B) {
```

Comment:
```suggestion
func BenchmarkHead_WalCommit(b *testing.B) {
```

All good benchmarks have varying input, don't they? 😅  I wouldn't make the test name too long because of that